modules/microshift-firewall-update-for-service.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-firewall.adoc
+
+
+:_content-type: CONCEPT
+[id="microshift-firewall-update-for-service_{context}"]
+= Updating firewall ports when a service is exposed
+
+Firewalld is often active when you run services on {microshift-short}. This can disrupt certain services on {microshift-short} because traffic to the ports might be blocked by the firewall. You need to ensure the necessary firewall ports are open if you want certain services to be accessible from outside the host. There are several options for opening your ports:
+
+* Services of the `NodePort` and `LoadBalancer` type are automatically available with OVN-K.
```

Comment:
I'm having trouble figuring out what the two options are with how these are written. I would try to make it clearer.

The first one talks about services of these two types. What about of the other types? And are they only available with OVN-K? (like, is there an option other than OVN-K that users could be using where these wouldn't be availabel automatically?

Reply:
I think they mainly just wanted to reference the two types. But I'm also pretty sure that OVN-K is automatically part of MicroShift the assumption is that the users use the services with OVN-K. 
Lemme ask about that though, becuase my experience with firewall ports is limited